src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -287,54 +289,58 @@ const ClientCardImage = ({
         onClose={handleClose}
         clientId={client.id}
       />
-      <Link
-        component='button'
-        underline='none'
-        onClick={handleOpen}
-        ref={linkRef}
-        sx={{
-          position: 'relative',
-          width: size,
-          height: size,
-          '&:focus-within > .overlay': {
-            opacity: 1,
-          },
-          '&:hover > .overlay': {
-            opacity: 1,
-          },
-        }}
-      >
-        <ClientCardImageElement size={size} client={client} />
-        {client.image?.base64 ? (
-          // Has photo
-          <Box
-            className='overlay'
-            sx={{
-              ...overlaySx,
-              backgroundColor: 'rgba(0,0,0, 0.5)',
-            }}
-          >
-            <Chip
-              label='Update Photo'
-              icon={<EditIcon />}
+      {client.access.canEditClient ? (
```

Comment:
GH is displaying this as a more complicated diff than it really is; the only difference is wrapping in the `{client.access.canEditClient ? (...) : ...` clause.